lib/std/process.zig

```diff
@@ -1067,87 +1067,6 @@ pub fn getBaseAddress() usize {
     }
 }
 
-/// Caller owns the result value and each inner slice.
```

Comment:
Why does that PR remove this?

Reply:
> Why does that PR remove this? 

This is separate commit, but I'll drop this if you want

Reply:
I just wanted to not create second CI run for this unrelated commit